app/models/payment/braintree_customer.rb

```diff
@@ -1,5 +1,9 @@
 class Payment::BraintreeCustomer < ActiveRecord::Base
   belongs_to :member
   has_many :braintree_payment_methods, class_name: Payment::BraintreePaymentMethod, primary_key: :customer_id, foreign_key: :customer_id
-  belongs_to :default_payment_method, class_name: Payment::BraintreePaymentMethod
+
+
+  def default_payment_method
+    braintree_payment_methods.order('created_at desc').first
+  end
```

Comment:
I think when we decide the latest payment method isn't what we want, then the best option would be a `default` boolean flag on `Braintree::PaymentMethod`, allowing the member to select their default.